src/common/prod/data.tf

```diff
@@ -107,4 +107,10 @@ data "azurerm_subnet" "itn_msgs_sending_func_snet" {
   name                 = "${local.project_itn}-msgs-sending-func-snet-01"
   resource_group_name  = local.core.networking.itn.vnet_common.resource_group_name
   virtual_network_name = local.core.networking.itn.vnet_common.name
-}
\ No newline at end of file
+}
+
+data "azurerm_subnet" "itn_pep_snet" {
+  name                 = "${local.project_itn}-pep-snet-01"
+  resource_group_name  = local.core.networking.itn.vnet_common.resource_group_name
+  virtual_network_name = local.core.networking.itn.vnet_common.name
+}
```

Comment:
I think you can get the pep snet id with:
`local.core.networking.itn.pep_snet.id`
```suggestion
```